go.mod

```diff
@@ -6,40 +6,31 @@ require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/cpuguy83/go-md2man v1.0.10 // indirect
 	github.com/creack/pty v1.1.7
-	github.com/digitalocean/godo v1.22.0
+	github.com/digitalocean/godo v1.26.0
 	github.com/dustin/go-humanize v1.0.0
 	github.com/fatih/color v1.7.0
 	github.com/gobwas/glob v0.2.3
 	github.com/golang/mock v1.3.1
-	github.com/google/gofuzz v0.0.0-20170612174753-24818f796faf // indirect
-	github.com/google/uuid v1.1.0
+	github.com/google/uuid v1.1.1
```

Comment:
why did uuid get upgraded in this PR as well? Can we not upgrade this unless it is a direct dependency?

Reply:
I'm not sure. I think this change was made when I added `k8s.io/api/core/v1` as a dependency. What do you suggest I do here?

Reply:
if it is, we can leave it but I wanted to understand why it got upgraded.